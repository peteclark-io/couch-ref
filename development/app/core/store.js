'use strict';

import { createStore } from 'redux';
import reducer from '../ducks/reducers';

const store = createStore(reducer);

export default store;
