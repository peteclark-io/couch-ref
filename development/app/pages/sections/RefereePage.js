'use strict';

import React from 'react';
import RefereeTable from '../../components/Referees/RefereeTable';

const RefereePage = React.createClass({

  render: function() {
    return (
      <RefereeTable />
    );
  }
});

export default RefereePage;
