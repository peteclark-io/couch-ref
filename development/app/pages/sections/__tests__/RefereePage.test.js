'use strict';

import React from 'react';
import RefereePage from '../RefereePage';
import {shallow} from 'enzyme';

import {Provider} from 'react-redux';
import store from '../../../core/store';

test('RefereePage section', () => {
   const rendered = shallow(
      <Provider store={store}>
         <RefereePage />
      </Provider>
   );

   let tree = rendered.debug();
   expect(tree).toMatchSnapshot();
});
