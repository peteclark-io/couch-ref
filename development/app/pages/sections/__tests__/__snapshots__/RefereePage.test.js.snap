exports[`test RefereePage section 1`] = `"<RefereePage />"`;
